source/kotlin-sync.txt

```diff
@@ -0,0 +1,146 @@
+==========================
+Kotlin Sync Driver (Alpha)
+==========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: twocols
+
+.. note:: Alpha Release
+
+   The Kotlin driver is currently in alpha release and available only for evaluation purposes,
+   to validate functionality, and to gather feedback.
+   It is not recommended for production deployments as we may introduce breaking changes.
+   This feature doesn't include formal consulting, SLAs, or technical support obligations.
+
+Introduction
+------------
+
+Welcome to the documentation site for the Kotlin Sync Driver, the
+official MongoDB driver for synchronous Kotlin applications. 
+
+- `API Documentation <{+kotlin-api+}/apidocs/mongodb-driver-kotlin-sync/>`__
+
+- `Source Code <https://github.com/mongodb/mongo-java-driver/tree/master/driver-kotlin-sync/>`__
+
+If you need to make asynchronous calls between your application and MongoDB with 
+coroutines, use the :driver:`MongoDB Kotlin Coroutine Driver </kotlin/coroutine/current/>` 
+instead of the Sync Driver.
+
+Installation
+------------
+
+The recommended way to get started using the driver in your project is with
+a dependency management system.
+
+If you are using `Gradle <https://gradle.org/>`__, add the following to your
+``build.gradle.kts`` dependencies list:
+
+.. code-block:: xml
+   :caption: build.gradle.kts
+
+   dependencies {
+    implementation("org.mongodb:mongodb-driver-kotlin-sync:4.9.1")
+   }
+
+If you are using `Maven <https://maven.apache.org/>`__, add the following to
+your ``pom.xml`` dependencies list:
+
+.. code-block:: xml
+   :caption: build.gradle.kts
+
+   <dependencies>
+      <dependency>
+         <groupId>org.mongodb</groupId>
+         <artifactId>mongodb-driver-kotlin-sync</artifactId>
+         <version>4.9.1</version>
+      </dependency>
+   </dependencies>
+
+.. _connect-atlas-kotlin-sync-driver:
+
+Connect to MongoDB Atlas
+------------------------
+
+.. include:: /includes/atlas-connect-blurb.rst
+
+.. code-block:: kotlin
+
+   import com.mongodb.ConnectionString
+   import com.mongodb.kotlin.client.sync.MongoClient
+   import com.mongodb.kotlin.client.sync.MongoDatabase
+   import com.mongodb.MongoClientSettings
+
+   // Replace the placeholder with your Atlas connection string
+   val uri = "mongodb+srv://<username>:<password>@<cluster-address>/test?w=majority"
```

Comment:
Question:
Could the placeholder be "\<connection string\>" instead, similar to how it is formatted on the Kotlin sync connection example?
Other drivers use that convention because the instructions mention retrieving the value from Atlas. This would also help clarify the text in lines 114-116:

> To test whether you can connect to your server, replace the connection
string in the :ref:`Connect to MongoDB Atlas <connect-atlas-kotlin-sync-driver>`
code example and run it.